packages/contracts-bedrock/scripts/Deployer.sol

```diff
@@ -427,6 +438,36 @@ abstract contract Deployer is Script {
         return string(res);
     }
 
+    /// @dev Pulls the `_initialized` storage slot information from the Forge artifacts for a given contract.
+    function getInitializedSlot(string memory _contractName) internal returns (StorageSlot memory slot_) {
+        string memory storageLayout = getStorageLayout(_contractName);
+
+        string[] memory command = new string[](3);
+        command[0] = Executables.bash;
+        command[1] = "-c";
+        command[2] = string.concat(
+            Executables.echo,
+            " '",
+            storageLayout,
+            "'",
```

Comment:
THis seems to be the convention, but why can't we just do this?
```suggestion
            " 'storageLayout'",
```

Reply:
That doesn't work, no? `storageLayout` is a variable, can't inline it into the string - we're passing the value in `storageLayout`, not a string "storageLayout" :)

Reply:
sidenote - wen string interpolation in solc